### DISCUSSION TITLE
About ui

### DISCUSSION THREAD

**Discussion by Parad0x84** [obsproject/obs-studio#4206] [General]
1-) Which library used here for creating ui and which library is best for creating ui with native c?
2-) Can you suggest me educational content for Qt or any library which easy to learn and best for small apps?

**Comment by WizardCM** [comment]
The OBS Studio UI is created using [Qt](https://www.qt.io/download-open-source)

**Reply by Parad0x84** [reply]
> The OBS Studio UI is created using [Qt](https://www.qt.io/download-open-source)

Can you suggest me educational content for Qt or any library which easy to learn and best for small apps?
